Resources/config/mongodb.xml

```diff
@@ -138,6 +139,8 @@
 
         <service id="doctrine_mongodb.odm.command_logger" class="Doctrine\Bundle\MongoDBBundle\APM\PSRCommandLogger" public="false">
             <argument type="service" id="logger" on-invalid="null" />
+            <argument>%doctrine_mongodb.odm.command_logger.prefix%</argument>
```

Comment:
@alcaeus: If this is now part of the service definition, would it make sense to change the default value in the constructor signature to `null`?

Reply:
I don't like using DI parameters for this, at least not to have this configurable to the outside. Instead, this should be configured using the configuration and then injected into the DI container by the extension.